### PR TITLE
Change label in doc issue template

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

CL-* is used only for ChangeLog automation.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [x] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Documentation_Issue.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Documentation Issue 📚
 about: File issues regarding documentation within the [PowerShell-Docs](https://github.com/MicrosoftDocs/PowerShell-Docs) repository
 title: "Documentation Issue"
-labels: CL-Docs
+labels: Issue-Question,Area-Documentation
```
Comment (RDIL) [inline .github/ISSUE_TEMPLATE/Documentation_Issue.md]: Not sure, but I think this might need to be an array or just 1 label.

Comment (vexx32) [reply .github/ISSUE_TEMPLATE/Documentation_Issue.md]: if I remember the syntax right this needs to look more like this:
```suggestion
labels: [Issue-Question, Area-Documentation]
```

Comment (iSazonov) [reply .github/ISSUE_TEMPLATE/Documentation_Issue.md]: Working sample https://raw.githubusercontent.com/lee-dohm/test-repo/master/.github/ISSUE_TEMPLATE/Bug_report.md

**Comment by iSazonov** [comment]
@TravisEz13 Can we merge the template update?

**Comment by TravisEz13** [comment]
restarted the windows CI.  Ping me if it passes.

**Comment by RDIL** [comment]
It failed

**Comment by vexx32** [comment]
Weren't there a couple PRs to remove appveyor refs from the build? Error is:
```
Invoke-AppveyorInstall : The term 'Invoke-AppveyorInstall' is not recognized as the name of a cmdlet, function, script 
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct 
and try again.
```

Maybe try fast forwarding your branch to latest master?

**Comment by RDIL** [comment]
Yeah, that would be the set of PRs I opened.  Rebasing (should) fix it

**Comment by iSazonov** [comment]
Rebased.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**